aten/src/ATen/native/quantized/cpu/conv_serialization.h

```diff
@@ -37,23 +39,45 @@
  *  2. list of optional tensors
  *    0: bias
  *
- *  Note: version is a string and conv params are packed into a Tensor
- *    to make ONNX happy (ints and containers of ints are not supported).
+ * Version 3
+ *
+ * - Fields:
+ *  0. version (int)
+ *  1. list of integer configuration values
```

Comment:
`int64_t`?

Reply:
Looks like this breaks in Visual C++ . :/